aiven/resource_flink_test.go

```diff
@@ -13,12 +13,17 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 )
 
-func TestAccAiven_flinkBasic(t *testing.T) {
-	resourceName := "aiven_flink.bar"
-	serviceName := fmt.Sprintf("test-acc-flink-%s", acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum))
+func TestAccAiven_flink(t *testing.T) {
+	t.Parallel()
```

Comment:
Looks like we do not need that since we use `resource.ParallelTest`

Reply:
I tested and it does not run in parallel without that! I think `resource.ParallelTest(tt)` only registers the test suit as parallel on `tt`.